src/internal/serviceenv/service_env.go

```diff
@@ -408,6 +410,8 @@ func (env *NonblockingServiceEnv) newDirectListener() col.PostgresListener {
 		dbutil.WithDBName(env.config.PostgresDBName),
 		dbutil.WithUserPassword(env.config.PostgresUser, env.config.PostgresPassword),
 		dbutil.WithSSLMode(env.config.PostgresSSL),
+		// Note, enabling query logs with WithQueryLog on the direct listener seems to break
```

Comment:
yikes

Reply:
I think this is because of how pgx generates a "fake" DSN to connect to, so we end up connecting to the wrong place when the DSN is shared.  I didn't have time to look into it, though, but the symptom is that isAuthEnabled returns the wrong answer, so I know it breaks listeners.

Reply:
should we have an action item to understand this behavior?

Reply:
I adjusted the note here.  It's understood, we use two different postgres libraries 😂 and only pgx can do logging.